nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -26,15 +23,14 @@ static NFCSTATUS phNciNfc_SendReset(void *pContext);
 static NFCSTATUS phNciNfc_CompleteReleaseSequence(void *pContext, NFCSTATUS wStatus);
 static NFCSTATUS phNciNfc_CompleteNfccResetSequence(void *pContext, NFCSTATUS wStatus);
 
-static NFCSTATUS phNciNfc_ResetNtfCb(void* pContext, void* pInfo, NFCSTATUS status);
+NFCSTATUS phNciNfc_ResetNtfCb(void* pContext, void* pInfo, NFCSTATUS status);
 
 static NFCSTATUS phNciNfc_RegAllNtfs(void* pContext);
```

Comment:
That's true i missed it was already present in phNciNfc_Init.h.
Will fix that as well